modules/async-icq/e2e/upgrade_test.go

```diff
@@ -180,7 +182,7 @@ func ValidatorVoting(t *testing.T, ctx context.Context, chain *cosmos.CosmosChai
 	err := chain.VoteOnProposalAllValidators(ctx, proposalID, cosmos.ProposalVoteYes)
 	require.NoError(t, err, "failed to submit votes")
 
-	_, err = cosmos.PollForProposalStatus(ctx, chain, height, height+haltHeightDelta, proposalID, cosmos.ProposalStatusPassed)
+	_, err = cosmos.PollForProposalStatusV8(ctx, chain, height, height+haltHeightDelta, proposalID, cosmos.ProposalStatusPassedV8)
```

Comment:
if baseChain is moved to `v7.X` this would use the standard `PollForProposalStatus`. 

_(future note: The next tag of interchaintest v8 will rename this to V1 and V1beta1)_